priv/static/common.css

```diff
@@ -0,0 +1,49 @@
+/* This file contains design tokens used in core Gleam projects */
+:root {
+  /* Greys */
+  --col-black: #000000;
+  --col-grey-950: #151515;
+  --col-grey-900: #1e1e1e;
+  --col-grey-800: #2f2f2f;
+  --col-grey-100: #f5f5f5;
+  --col-azure-900: #202431;
+  --col-azure-800: #292d3e;
+  --col-azure-600: #51597b;
+  --col-azure-500: #7e818b;
+  --col-sepia-450: #b1a894;
+  --col-sepia-400: #cac0a9;
+  --col-sepia-300: #e3d8be;
+  --col-sepia-100: #fffbe8;
+  --col-sepia-050: #fefefc;
+
+  /* Colors */
+  --col-faff-pink-800: #584355;
+  --col-faff-pink-300: #ffaff3;
+  --col-boi-blue-200: #a6f0fc;
+  --col-menthol-200: #c8ffa7;
+  --col-caramel-300: #ffd596;
+  --col-deep-saffron-400: #ff9d35;
+  --col-tomato-400: #ff6262;
+
+  /* Semantic colors */
+  --brand-primary: var(--col-faff-pink-300);
+  --brand-primary-dim: var(--col-faff-pink-800);
```

Comment:
Remove these two please. It is clearer to use the names and we are not going to be swapping out these colours 🙏

Reply:
done